frontend/src/components/Icons.tsx

```diff
@@ -120,6 +120,28 @@ export const InfoTriangleIcon = ({
   );
 };
 
+export const RoundedInfoTriangleIcon = ({
```

Comment:
Heh... Just wondering if maybe we should ask design if all existing instances of `InfoTriangleIcon` should also be rounded - I can't think of a reason why some should be rounded and some should not.